src/components/App.js

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import './App.css';
 
+import ButtonPanel from './ButtonPanel';
+
 function App() {
   return (
     <div className="App">
-      <h1> Welcome to React Calculator </h1>
+      <ButtonPanel/>
     </div>
```

Comment:
This should also have the `Display` Component see [requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/Display&Button.png)